packages/compile-solidity/test/test_supplier.js

```diff
@@ -45,19 +45,19 @@ describe("CompilerSupplier", function() {
         "utf-8"
       );
 
-      const version6Pragma = await fse.readFile(
```

Comment:
Seems like we continue testing the Version 6 sources, no?

Reply:
Well, we never really explicitly tested version 6.  It looks like basically all our tests remained on 5.  The only thing updated to 6 was the test of native solc, which had to be.  And that's all I've updated here.  I figure updating other tests like they should be can be left to a later PR, presumably one written by someone who knows this code better...

Reply:
I'm not entirely upset if we skip 0.6, since there's a much larger effort of revisiting the way we organize these cases. Currently it's all ad hoc and kind of a mess.

Although I could be convinced that we need to revisit our tests for 0.6! But that's definitely a separate issue than this PR.